cmd/regex_format.go

```diff
@@ -326,16 +326,48 @@ func findUpperCaseOnIgnoreCaseFlag(lines []string, iFlag bool) (bool, string) {
 	// check if the file contains uppercase letters
 	if iFlag {
 		for number, line := range lines {
-			// ignore line if it starts with ##!
-			if strings.HasPrefix(line, "##!") {
+			// ignore line if it starts with the following regex: \s+##!
+			if regex.CommentRegex.MatchString(line) {
```

Comment:
Definition directives should not be ignored: `##!> define something "AbC-`.

Reply:
Ugh, parsing definitions could be a rabbit hole. Let me see if there is a smarter way 🤔

Reply:
Found a solution, but I would say that it feel we are repeating things from the parser here... 🤷

Reply:
Ok, updating the global definition regex looks like it makes it better now.